### PR TITLE
Fix Set-Location examples.

### PR DESCRIPTION
**Pull request by sethvs**
# PR Summary
Fix `Set-Location` examples.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Management/Set-Location.md [hunk 1]
```diff
@@ -69,27 +69,37 @@ Env:\
...
 
 ### Example 3: Set location to the C: drive
 
 ```powershell
-PS C:\> Set-Location C:
+PS Env:\> Set-Location C:\
 ```
 
-This command sets the current location C: drive in the FileSystem provider.
+This command sets the current location to the root of the `C:` drive in the FileSystem provider.
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Management/Set-Location.md:81]: Do you think we should change this one? It highlights that setting the location without specifically defining the location will push the location to the current location in the FileSystem provider.  I think from the separate provider change is good though so something like:

`PS Env:\> Set-Location -Path C:`

Comment (sethvs) [reply reference/5.1/Microsoft.PowerShell.Management/Set-Location.md]: I get the point.
Maybe like this 91812d0d4863a5ab8e4fbbffb899299c504378ad?

File: reference/6/Microsoft.PowerShell.Management/Set-Location.md [hunk 2]
```diff
@@ -70,14 +70,20 @@ PS Env:\>
...
+PS C:\Windows\> Set-Location HKLM:\
+PS HKLM:\> Set-Location C:
+PS C:\Windows\>
 ```
 
-This command sets the current location to the root of the `C:` drive in the FileSystem provider.
+The first command sets the current location to the root of the `HKLM:` drive in the Registry
+provider.
+The second command sets the current location to the `C:` drive in the FileSystem provider.
+When the drive name is specified in the form `DriveName:` (without backslash), then the cmdlet sets
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Management/Set-Location.md:84]:
```suggestion
When the drive name is specified in the form `<DriveName>:` (without backslash), the cmdlet sets
```

File: reference/6/Microsoft.PowerShell.Management/Set-Location.md [hunk 3]
```diff
@@ -70,14 +70,20 @@ PS Env:\>
...
+the current location to the current location in the PSDrive.
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Management/Set-Location.md:85]:
```suggestion
the location to the current location in the PSDrive.
```

File: reference/6/Microsoft.PowerShell.Management/Set-Location.md [hunk 4]
```diff
@@ -70,14 +70,20 @@ PS Env:\>
...
-PS Env:\> Set-Location C:\
-PS C:\>
+```powershell
+PS C:\Windows\> Set-Location HKLM:\
+PS HKLM:\> Set-Location C:
+PS C:\Windows\>
 ```
 
-This command sets the current location to the root of the `C:` drive in the FileSystem provider.
+The first command sets the current location to the root of the `HKLM:` drive in the Registry
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Management/Set-Location.md:81]:
```suggestion
The first command sets the location to the root of the `HKLM:` drive in the Registry
```

File: reference/6/Microsoft.PowerShell.Management/Set-Location.md [hunk 5]
```diff
@@ -70,14 +70,20 @@ PS Env:\>
...
+provider.
+The second command sets the current location to the `C:` drive in the FileSystem provider.
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Management/Set-Location.md:83]:
```suggestion
The second command sets the location to the current location of the `C:` drive in the FileSystem provider.
```

File: reference/6/Microsoft.PowerShell.Management/Set-Location.md [hunk 6]
```diff
@@ -70,14 +70,20 @@ PS Env:\>
...
+When the drive name is specified in the form `DriveName:` (without backslash), then the cmdlet sets
+the current location to the current location in the PSDrive.
+To get current location in the PSDrive use the `Get-Location` cmdlet with the **PSdrive** parameter.
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Management/Set-Location.md:86]:
```suggestion
To get the current location in the PSDrive use `Get-Location -PSDrive <DriveName>`.
```

**Comment by chasewilson** [comment]
Thanks for the contribution as well as your patience in the process! Everything looks good to me and as soon as the final build is done, I'll merge. Great job :)

**Pull request merged**